niyobobo/PoliticoReact
# #167270716 Setup React project

### What does this PR do?
Setup react project
### Description of Task to be completed?
- Configure webpack
- Configure jest for running test
- Setup react project
- Configure TravisCI
- Configure Coverall
### How should this be manually tested?
- Clone the repository locally with `git clone git@github.com:niyobobo/PoliticoReact.git` command.
- Check out to this branch by running `git checkout ch-initialise-project-167270716`.
- Install dependencies by running `npm install`.
- To start the server run `npm start`, you will be directed to this URL`http://localhost:8000/` in your browser.
### Any background context you want to provide?
N/A
### What are the relevant pivotal tracker stories?
N/A
### Questions:
N/A
